Traceback (most recent call last):
  File "c:\Users\Admin\miniconda3\envs\posformer\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\Users\Admin\miniconda3\envs\posformer\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "c:\Users\Admin\.vscode\extensions\ms-python.debugpy-2024.10.0-win32-x64\bundled\libs\debugpy\__main__.py", line 39, in <module>
    cli.main()
  File "c:\Users\Admin\.vscode\extensions\ms-python.debugpy-2024.10.0-win32-x64\bundled\libs\debugpy/..\debugpy\server\cli.py", line 430, in main
    run()
  File "c:\Users\Admin\.vscode\extensions\ms-python.debugpy-2024.10.0-win32-x64\bundled\libs\debugpy/..\debugpy\server\cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "c:\Users\Admin\.vscode\extensions\ms-python.debugpy-2024.10.0-win32-x64\bundled\libs\debugpy\_vendored\pydevd\_pydevd_bundle\pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "c:\Users\Admin\.vscode\extensions\ms-python.debugpy-2024.10.0-win32-x64\bundled\libs\debugpy\_vendored\pydevd\_pydevd_bundle\pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "c:\Users\Admin\.vscode\extensions\ms-python.debugpy-2024.10.0-win32-x64\bundled\libs\debugpy\_vendored\pydevd\_pydevd_bundle\pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "x:\Repositories\PosStateFormer\train.py", line 101, in <module>
    train(config)
  File "x:\Repositories\PosStateFormer\train.py", line 71, in train
    checkpoint_callback = ModelCheckpoint(save_top_k=config.trainer.callbacks[1].init_args.save_top_k,
  File "c:\Users\Admin\miniconda3\envs\posformer\lib\site-packages\pytorch_lightning\callbacks\model_checkpoint.py", line 246, in __init__
    self.__init_monitor_mode(mode)
  File "c:\Users\Admin\miniconda3\envs\posformer\lib\site-packages\pytorch_lightning\callbacks\model_checkpoint.py", line 455, in __init_monitor_mode
    torch_inf = torch.tensor(np.Inf)
  File "c:\Users\Admin\miniconda3\envs\posformer\lib\site-packages\numpy\__init__.py", line 414, in __getattr__
    raise AttributeError(
AttributeError: `np.Inf` was removed in the NumPy 2.0 release. Use `np.inf` instead.
